src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+.wrapper {
+  margin-top: 20px;
+}
+
+.searchBarWrapper {
+  position: relative;
+  width: 100%;
+}
+
+.searchBar {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  transition: box-shadow 150ms;
+  color: #3d4e61;
+  font-family: "Avenir", Arial, sans-serif;
+  box-sizing: border-box;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  transition: box-shadow 150ms;
+}
+
+.searchBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.searchBar.barSizeL {
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  font-weight: 900;
+}
+
+.searchBar.barSizeL:focus {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-family: "Avenir", Arial, sans-serif;
+  font-weight: 900;
+}
+
+.searchBar.barSizeS {
+  position: relative;
+  height: 42px;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.searchBar::placeholder {
+  position: absolute;
+  color: #3d4e61;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.searchBar.barSizeL::placeholder {
+  width: 158px;
+  left: 62px;
+  top: 35.71%;
+  bottom: 37.14%;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.searchBar.barSizeL:focus::placeholder {
+  text-shadow: none;
+}
+
+.searchBar.barSizeS::placeholder {
+  left: 33px;
+  top: 12px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.searchBar.barSizeS:hover::placeholder {
+  text-shadow: none;
+  transition: text-shadow 150ms;
+}
+
+.searchIcon {
```

Comment:
for icon try to use pseudo - element for label.
It is just decor and not necessary make more elements in html for that.